data/Makefile.am

```diff
@@ -39,8 +39,34 @@ install-sample-data:
 	rm fedora-20-icons.tar.gz; \
 	cd -
 
+com.endlessm.SystemExternalApps.rules: com.endlessm.SystemExternalApps.rules.in
```

Comment:
I wonder if we shouldn't use the `org.gnome.Software` namespace (won't block PR on it tho)